src/common/ipc.ts

```diff
@@ -80,6 +80,6 @@ export function unsubscribeAllFromBroadcast(channel: string) {
 
 export function bindBroadcastHandlers() {
   handleRequest(subFramesChannel, async () => {
-    return toJS(await getSubFrames(), { recurseEverything: true });
+    return await getSubFrames(), { recurseEverything: true };
```

Comment:
Shouldn't it be just `return getSubFrames()`?

Reply:
Oh yes, I was just moving pieces too fast. Thanks for noticing 👍 (it's now fixed)